text/0000-dynamic-import-hints.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Dynamic Import Hints and Its Metadata
+status: ACTIVE
+
+created_at: 2020-03-16
+updated_at: 2020-03-16
+---
+
+# Hints Authoring Syntax and Its Metadata
+
+## Summary
+This RFC suggests authoring syntax for specifying dynamic import hints and describes its metadata shape.
+
+## Motivation
```

Comment:
I personally would like to see more explanation as to how the dynamic loading lifecycle in a typical application is improved with the hint. How would it typically be used and what gains could one expect?

Reply:
Very fair point.